ab_tool/views/experiment_pages.py

```diff
@@ -183,3 +192,28 @@ def track_selection_csv(request, experiment_id):
     course_id = get_lti_param(request, "custom_canvas_course_id")
     experiment = Experiment.get_or_404_check_course(experiment_id, course_id)
     return get_track_selection_csv(request, experiment, "track_selection.csv")
+
+@lti_role_required(ADMINS)
+def upload_track_assignments(request, experiment_id):
+    course_id = get_lti_param(request, "custom_canvas_course_id")
+    experiment = Experiment.get_or_404_check_course(experiment_id, course_id)
+    uploaded_file = request.FILES["track_assignments"]
+    if uploaded_file.size > MAX_FILE_UPLOAD_SIZE:
+        raise FILE_TOO_LARGE
+    uploaded_text = uploaded_file.read()
+    unassigned_students = get_unassigned_students(request, experiment)
+    students, errors = parse_uploaded_file(
+            experiment, unassigned_students, uploaded_text, uploaded_file.name
+    )
+    if errors:
+        return render_to_response("ab_tool/spreadsheetErrors.html", {"errors": errors})
+    
+    for student_id, track in students.items():
+        ExperimentStudent.objects.create(
+            student_id=student_id, course_id=experiment.course_id,
+            track=track, lis_person_sourcedid=None,
```

Comment:
lis_person_sourcedid here is None because we are creating students off of data from the SDK (which doesn't provide this value).  While lis_person_sourcedid is the same as sis_id in all the examples I've seen, I thought it was better to set it to None since we don't know it for certain.  I'll add a comment